spring-components/pom.xml

```diff
@@ -20,6 +20,13 @@
 	<description>Components to use with Spring</description>
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
+				<groupId>org.junit</groupId>
+				<artifactId>junit-bom</artifactId>
+				<version>${junit.version}</version>
+				<type>pom</type>
```

Comment:
Does this need to be included here if it's already in the root pom?

Reply:
The spring one overrides it.